source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/customcontrols/ValidatedAutoCompleteTextView.java

```diff
@@ -0,0 +1,112 @@
+package io.adaptivecards.renderer.input.customcontrols;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.content.res.Resources;
+import android.util.TypedValue;
+import android.widget.AutoCompleteTextView;
+import android.widget.LinearLayout;
+
+import androidx.core.content.ContextCompat;
+
+import io.adaptivecards.R;
+
+/**
+ * ValidatedAutoCompleteTextView is an extension of AutoCompleteTextView that supports the usage of
+ * the adaptive_state_input_invalid boolean flag defined in attrs.xml for using a custom background drawable
+ */
+@SuppressLint("AppCompatCustomView")
```

Comment:
It seems to be recommended to extend [`AppCompatAutocompleteTextView`](https://developer.android.com/reference/androidx/appcompat/widget/AppCompatAutoCompleteTextView) for back-compat, which might fix this warning